config/zones/FI.yaml

```diff
@@ -182,13 +191,19 @@ emissionFactors:
         source: EU-ETS, ENTSO-E 2022; Oberschelp, Christopher, et al. "Global emission
           hotspots of coal power generation."
         value: 848.9178430000001
+      - datetime: '2023-01-01'
+        source: EU-ETS, ENTSO-E 2023
+        value: 778.07
     gas:
       - datetime: '2021-01-01'
         source: EU-ETS, ENTSO-E 2021; IPCC 2014
         value: 415.9885661
       - datetime: '2022-01-01'
         source: EU-ETS, ENTSO-E 2022; IPCC 2014
         value: 415.99
+      - datetime: '2023-01-01'
+        source: EU-ETS, ENTSO-E 2023
+        value: 596.61
```

Comment:
The gas power plant _Vuosaaren B-voimalaitos_ is causing the significant increase in emission factor. Emissions have increased from 87,269 in 2022 to 120,283 in 2023, while ETS allocations have decreased from 85,191 to 31,848.